submarine-server/server-submitter/submitter-k8s/src/main/java/org/apache/submarine/server/submitter/k8s/K8sJobSubmitter.java

```diff
@@ -116,4 +127,43 @@ CustomResourceJobList listCustomResourceJobs(K8sJobRequest request) {
     }
     return null;
   }
+
+  @VisibleForTesting
+  void createCustomResourceDefinition(String filePath) {
```

Comment:
Is this method added for the future use? We'd better add UT for this method like others

Reply:
Nowadays it's unused, so  I'll remove these methods.